stacks/python/3.0.0/devfile.yaml

```diff
@@ -29,7 +29,7 @@ components:
       mountSources: true
       endpoints:
         - name: http-python
-          targetPort: 8080
+          targetPort: 8081
```

Comment:
@maysunfaisal I agree. As we would like in the future to use `8080` instead of `8081` I would suggest too to update the rest of the `8081` ports to `8080`. I've tested locally an example repo https://github.com/thepetk/test_devfile_python_sample and seems to work fine.